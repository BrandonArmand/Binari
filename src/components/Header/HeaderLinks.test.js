import React from "react";
import { BrowserRouter as Router } from "react-router-dom/cjs/react-router-dom.min";
import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import HeaderLinks from "./HeaderLinks";


describe("HeaderLinks", () => {
    it("renders snapshot", () => {
        const { container } = render(<Router><HeaderLinks /></Router>);
        expect(container).toMatchSnapshot();
    });
    it("renders HeaderLinks", () => {
        render(
            <Router>
                <HeaderLinks />
            </Router>
        );
        expect(screen.getByText("Playground")).toBeInTheDocument();
        expect(screen.getByText("About")).toBeInTheDocument();
    });
});
