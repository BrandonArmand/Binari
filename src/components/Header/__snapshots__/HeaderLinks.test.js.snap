// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderLinks renders snapshot 1`] = `
<div>
  <ul
    class="MuiList-root makeStyles-list-1 MuiList-padding"
  >
    <li
      class="MuiListItem-root makeStyles-listItem-2 MuiListItem-gutters"
    >
      <a
        class="makeStyles-link-10"
        href="/playground"
      >
        <button
          class="MuiButtonBase-root MuiButton-root makeStyles-button-14 makeStyles-transparent-28 makeStyles-navLink-4 MuiButton-text"
          tabindex="0"
          target="_blank"
          title="Get Started"
          type="button"
        >
          <span
            class="MuiButton-label"
          >
            Playground
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </a>
    </li>
    <li
      class="MuiListItem-root makeStyles-listItem-2 MuiListItem-gutters"
    >
      <a
        class="makeStyles-link-10"
        href="/about"
      >
        <button
          class="MuiButtonBase-root MuiButton-root makeStyles-button-14 makeStyles-transparent-28 makeStyles-navLink-4 MuiButton-text"
          tabindex="0"
          target="_blank"
          title="Get to know me!"
          type="button"
        >
          <span
            class="MuiButton-label"
          >
            About
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </a>
    </li>
    <li
      class="MuiListItem-root makeStyles-listItem-2 MuiListItem-gutters"
    >
      <a
        aria-disabled="false"
        class="MuiButtonBase-root MuiButton-root makeStyles-button-14 makeStyles-transparent-28 makeStyles-navLink-4 MuiButton-text"
        href="https://github.com/BrandonArmand/Binari"
        tabindex="0"
        target="_blank"
        title="Checkout my GitHub"
      >
        <span
          class="MuiButton-label"
        >
          <i
            class="makeStyles-socialIcons-9 fab fa-github"
          />
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </a>
    </li>
  </ul>
</div>
`;
